torchbenchmark/models/hf_GPT2/requirements.txt

```diff
@@ -0,0 +1,3 @@
+transformers==4.5.0
+datasets
```

Comment:
is this package,  `datasets`  used? We seem to be generating all the inputs ourselves?

Reply:
not totally sure... these HF models do some weird imports internally. I didn't think we needed SentencePiece either, but the BigBird model imports it somewhere internally.